### PR TITLE
Add minimal footer

### LINKED ISSUE #605: Consider adding a minimal footer on every page
During user testing, we got feedback about not having some info that people need.  For example: who to contact if there is trouble?  Who made this?  Where can you make feedback?, etc.

We could consider adding a minimal footer to the bottom of the design.  See https://chakra-templates.dev/page-sections/footer for some ideas.  A lot of these are too large (we don't have tons of links).  However, we could include:

- Link to GitHub project
- ITS Acceptable Use Policy Link
- Contact Us email link

etc

### PR DESCRIPTION
**Pull request by SerpentBytes**
## Description
Closes #605.

### Demo

https://user-images.githubusercontent.com/50856799/233436441-74080813-3760-457a-830e-5648c70cad60.mov

### CODE REVIEW

**Comment by SerpentBytes** [comment]
The footer is a little buggy on some views and smaller screen resolutions. At the moment, I am adding `height` to be`100vh` on the container above the Footer, maybe that's the problem. I am looking into this.

**Comment by humphd** [comment]
What I would do is use `<Flex>` to include the header, body, footer.  The header and footer will have fixed sizes, but the main part can use `flex="1"` to stretch to fill the whole space.  The `Flex` can have a `minHeight="100vh"` or `"100%` so it fills the entire viewport, and then use `flexDirection="column"` to have it spread things vertically.  This way, the main part of the page will use all the available space, and push the footer down to the bottom.

**Comment by SerpentBytes** [comment]
@humphd let me know if I got it right.

File: app/routes/_auth.tsx [hunk 1]
```diff
@@ -1,19 +1,21 @@
...
+import Footer from '~/components/footer';
 
 export default function Index() {
   return (
-    <Box>
+    <Flex flexDir="column">
       <Header />
       <main>
-        <Container maxW="container.xl" px={{ base: 4, md: 45 }}>
+        <Container flex="1" maxW="container.xl" px={{ base: 4, md: 45 }} minHeight="100vh">
```
Comment (humphd) [inline app/routes/_auth.tsx:13]: For `flex="1"` to work, you need to put it on a direct child of the `<Flex>` parent.  Can you eliminate `main`?  If not, use `<chakra.main flex="1">`

**Review by humphd — CHANGES_REQUESTED** [review]

**Review by humphd — CHANGES_REQUESTED** [review]
I have to scroll down an inch or so to get the footer to show:

Screenshot 2023-04-21 at 11 50 04 AM
Screenshot 2023-04-21 at 11 50 11 AM

It looks like it's roughly the hight of the header that's causing the issue.

I played with it locally, and this fixes it:

```diff
diff --git a/app/routes/_auth.tsx b/app/routes/_auth.tsx
index e59a8e7..4f219ab 100644
--- a/app/routes/_auth.tsx
+++ b/app/routes/_auth.tsx
@@ -7,9 +7,9 @@ import Footer from '~/components/footer';
 
 export default function Index() {
   return (
-    <Flex flexDir="column">
+    <Flex flexDir="column" h="100%">
       <Header />
-      <chakra.main flex="1" minHeight="100vh">
+      <chakra.main flex="1">
         <Container maxW="container.xl" px={{ base: 4, md: 45 }}>
           <Outlet />
         </Container>
```

**Comment by SerpentBytes** [comment]
@humphd I have updated the code with your fix.

**Review by humphd — APPROVED** [review]

**Review by sfrunza13 — APPROVED** [review]

**Pull request merged**